tests/specs/api/appUsers.spec.js

```diff
@@ -1,23 +1,21 @@
 import * as httpMock from '../../mocks/http';
+import { getAuthenticationHeaders } from '../../../src/utils/auth';
 
 
 describe('AppUsers API', () => {
   const serviceUrl = 'http://some-url.com';
   const userId = 'user-id';
-  const auth = {
+  const httpHeaders = getAuthenticationHeaders({
     appToken: 'token'
-  };
+  });
+
   let httpSpy;
   let api;
 
   beforeEach(() => {
     httpSpy = httpMock.mock();
     let AppUsersApi = require('../../../src/api/appUsers').AppUsersApi;
```

Comment:
I'll check if it change anything but I had the impression I would need to mock the http part before requiring the class... to be tested.